WordPress/src/jalapeno/res/drawable/app_icon_background.xml

```diff
@@ -0,0 +1,13 @@
+<vector
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:height="108dp"
+    android:width="108dp"
+    android:viewportHeight="108.0"
+    android:viewportWidth="108.0" >
+
+    <path
+        android:fillColor="#ff292f33"
```

Comment:
```suggestion
        android:fillColor="#ff77b255"
```

In order to differentiate `jalapeno` and `wasabi` flavors, I would change the `jalapeno` background to `#77b255` to look like a jalapeno.

Reply:
I added a red dot to the jalapeno icon. However, I agree that this change is even better ;).